src/controllers/controller.line.js

```diff
@@ -283,11 +283,19 @@ module.exports = function(Chart) {
 			var points = meta.data || [];
 			var area = chart.chartArea;
 			var ilen = points.length;
+			var dataset = me.getDataset();
+			var lineElementOptions = chart.options.elements.line;
+			var halfBorderWidth = helpers.valueOrDefault(dataset.borderWidth, lineElementOptions.borderWidth) / 2;
```

Comment:
We should not recompute the border width in `draw`, it's already done in `update()`.

I think `meta.dataset._model.borderWidth` is more accurate:

```javascript
var halfBorderWidth = (meta.dataset._model.borderWidth || 0) / 2;
```